etcd/datadog_checks/etcd/metrics.py

```diff
@@ -39,6 +39,11 @@
     'etcd_grpc_proxy_cache_misses_total': 'grpc.proxy.cache.misses.total',
     'etcd_grpc_proxy_events_coalescing_total': 'grpc.proxy.events.coalescing.total',
     'etcd_grpc_proxy_watchers_coalescing_total': 'grpc.proxy.watchers.coalescing.total',
+    'etcd_mvcc_db_total_size_in_bytes': 'mvcc.db.total.size.in_bytes',
+    'etcd_mvcc_delete_total': 'mvcc.delete.total',
+    'etcd_mvcc_put_total': 'mvcc.put.total',
+    'etcd_mvcc_range_total': 'mvcc.range.total',
+    'etcd_mvcc_txn_total': 'mvcc.txn.total',
```

Comment:
I think if we're trying to map these back to the pre-existing metric, the mapped metric name should be prefixed `debugging.`

https://github.com/DataDog/integrations-core/blob/f4388bf5bfdc97bd123f2155dd05341700e41008/etcd/datadog_checks/etcd/metrics.py#L13

Reply:
Hi @ChristineTChen, I did not know that we could map 2 or more metrics with the same name. Is that what you mean? I only followed the pattern that seems to be used by other metrics: `type.metric.range` (just guessing from all the other metrics in this file). 

In that way, I did not deleted the old mapping, only added a new one for the renaming to keep it compatible with previous etcd versions.

Reply:
Was the only difference in these metrics a name change? E.g if the metric type is the same as before and the metric is measuring the same thing, then we should map the metrics to the same name in datadog (if you were to upgrade etcd, you would expect not to have to change your dashboards/monitors).

Reply:
@ChristineTChen done!